client/src/components/existing-request/existing-request-edit.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <MainContainer>
     <template v-slot:container-header>
-      <v-col cols="auto" class="h4 py-0">
-       {{ editModeHeader }}
+      <v-col cols="auto" class="font-weight-bold h5 py-0">
+        {{ editModeHeader }}
```

Comment:
Maybe an extra space here

Reply:
I think this is correct? The old code had a single space indent, but we use 2 spaces everywhere else.